src/qlever/commands/benchmark_queries.py

```diff
@@ -227,41 +235,53 @@ def sparql_query_type(self, query: str) -> str:
             return "UNKNOWN"
 
     @staticmethod
-    def construct_get_queries_cmd(
-        queries_file: str, query_ids: str, query_regex: str, ui_config: str
-    ) -> str:
+    def filter_tsv_queries(
+        tsv_queries: list[str], query_ids: str, query_regex: str
+    ) -> list[str]:
         """
-        Construct get_queries_cmd from queries_tsv file if present or use
-        example queries by using ui_config. Use query_ids and query_regex to
-        filter the queries
+        Given a tab-separated list of queries, filter them and keep the
+        ones which are a part of query_ids or match with query_regex
```

Comment:
The input should be a list of tuples, see below. Change name to `filter_queries`